src/router/index.ts

```diff
@@ -1,14 +1,24 @@
-import { createRouter, createWebHistory } from 'vue-router'
+import { createRouter, createWebHistory } from "vue-router";
 
 const router = createRouter({
   history: createWebHistory(import.meta.env.BASE_URL),
   routes: [
     {
-      path: '/search',
-      name: 'search',
-      component: () => import('../views/SearchView.vue')
-    }
-  ]
-})
+      path: "/search",
+      name: "search",
+      component: () => import("../views/SearchView.vue"),
+    },
+    {
+      path: "/login",
```

Comment:
lets make that login will be the base (home page) as for now so we can add empty path to it- then when we type only localhost:3000 (without any extra path) we will get the login page. also can you change it for full screen? Now its next to the side menu which should only be seen when user is logged in :)
<img width="1167" alt="Screenshot 2022-05-07 at 11 14 53" src="https://user-images.githubusercontent.com/71661786/167247767-2350cf8b-1d04-4667-94ec-5d3d10809bb9.png">